test/pdfs/jbig2_huffman_2.pdf.link

```diff
@@ -0,0 +1 @@
+http://library.sciencemadness.org/library/books/gergel_isopropyl_bromide.pdf
```

Comment:
Please link to the Internet Archive instead, to avoid the file changing or becoming unavailable: https://web.archive.org/web/20170924053030/http://library.sciencemadness.org/library/books/gergel_isopropyl_bromide.pdf